backend/src/repositories/BlocksAuditsRepository.ts

```diff
@@ -106,6 +108,10 @@ class BlocksAuditRepositories {
         rows[0].acceleratedTxs = JSON.parse(rows[0].acceleratedTxs);
         rows[0].template = JSON.parse(rows[0].template);
 
+        if (!rows[0].firstSeen) {
```

Comment:
since `firstSeen` is a string here, a null timestamp of "0.000000" is still truthy and never gets deleted.